### ISSUE TITLE
Microphone tattler detects Bluetooth phone playing music as microphone

### ISSUE THREAD

**Issue by GranPC** [elementary/wingpanel-indicator-sound#103]
This happens because according to `pavucontrol`, my phone is an input device and playback is happening through a loopback stream.

![Screenshot from 2019-06-30 20-10-34](https://user-images.githubusercontent.com/1385480/60400382-56201c00-9b73-11e9-864f-524bb68a1b41.png)

I thought of filtering out any input devices that are either monitors or owned by `module-bluez5-device`, but I don't know whether that would also break the tattler for Bluetooth microphones.

**Comment by GranPC** [comment]
I have a branch that fixes this issue, let me know if you like my approach and I will create a pull request. https://github.com/GranPC/wingpanel-indicator-sound/commit/d121650acb8dfdad135226ae9895aabc9345c428

**Comment by cassidyjames** [comment]
@GranPC yeah I think open a Pull Request for visibility and then the team can review it.

**Comment by GranPC** [comment]
Done!
